apps/rush-lib/src/cli/utilities/base/BasePackage.ts

```diff
@@ -61,7 +61,7 @@ export class BasePackage {
   /**
    * The "version" field from package.json
```

Comment:
>The "version" field from package.json [](start = 5, length = 37)

Update comment.

The "version" field from package.json. This is expensive to read because we have to open the package.json file.  Only when DEBUG=true